python/dgl/graphbolt/minibatch.py

```diff
@@ -474,6 +474,49 @@ def node_pairs_with_labels(self):
         else:
             return None
 
+    def to_pyg_data(self):
+        """Construct a PyG Data from `MiniBatch`. This function only supports
+        node classification task on a homogeneous graph and the number of
+        features cannot be more than one.
+        """
+        from torch_geometric.data import Data
+
+        if self.sampled_subgraphs is None:
+            edge_index = None
+        else:
+            col_nodes = []
+            row_nodes = []
+            for subgraph in self.sampled_subgraphs:
```

Comment:
pyg train on multi-layers simultaneously as we are merging all subgraphs into single one?

Reply:
Yes pyg will merge the layers and unique all the edges